index.html

```diff
@@ -0,0 +1,145 @@
+                        <li class="p-cta"><a href="#">Sponsor</a></li>
+                        <li class="p-cta"><a href="#">News</a></li>
+                        <li class="p-cta call2action"><a href="#">Start Now</a></li>
+                    </ul>
+                </div>
+            </nav>
+        </header>
+        <div class="container">
+            <div class="headline">
+                <p class="intro font-face">Welcome to "ITalk"</p>
+                <h1 class="title font-face">Where we discuss matters that matters to us</h1>
+                <p class="discription font-face">Stay up to date with matters that affect young people, trending skills that are in demand all over the world :)</p>
+                <p class="date font-face">LEARN EVERYDAY + LIFE TIME SUPPORT</p>
+                <p class="location font-face">We are fully online | Support team available 24 hours |</p>
+            </div>
+        </div>
+    </section>
+    <section class="main-program">
+        <div class="program-bg"></div>
+        <div class="p-cont">
+            <p class="program-title">Main Program</p>
+            <hr>
+        </div>
+        
+        <div class="container wrapper">
+            <div class="card">
+                <div class="col-6 card-title">
+                    <img class="p-icon" src="images/lecturer.png" alt="lecture image">
+                    <p class="t1">Lecture</p>
+                </div>
+                <p class="program-disc">Watch some of our meterials from various expect across the world.</p>
+            </div>
+            <div class="card">
+                <div class="col-6 card-title">
+                    <img class="p-icon" src="images/board.png" alt="lecture image">
+                    <p class="t1">Exhibition</p>
+                </div>
+                <p class="program-disc">Our board contain timetables and announcements.</p>
+            </div>
+            <div class="card">
+                <div class="col-6 card-title">
+                    <img class="p-icon" src="images/chat.png" alt="lecture image">
+                    <p class="t1">Forum</p>
+                </div>
+                <p class="program-disc">Have questions, drop us as chat.</p>
+            </div>
+            <div class="card">
+                <div class="col-6 card-title">
+                    <img class="p-icon" src="images/workshop.png" alt="lecture image">
+                    <p class="t1">Mentors</p>
+                </div>
+                <p class="program-disc">You are assigned a personal mentor</p>
+            </div>
+            <div class="card">
+                <div class="col-6 card-title">
+                    <img class="p-icon" src="images/meeting.png" alt="lecture image">
+                    <p class="t1">Meetings</p>
+                </div>
+                <p class="program-disc">We have weekly meetings to address challenges we are faced.</p>
+            </div>
+        </div>
```

Comment:
- [ ] On mobile, there is a layout this section is supposed to follow. 
-  The middle section individual articles have their elements organized in an horizontal row, with the image at the left, title in the center, and description at the right. At this moment, you layout is different from what is on the guiding template. Kindly look into that and fix it.

### Your Design

![your-main-program-section](https://user-images.githubusercontent.com/26597432/197185249-c5f3ef21-200e-463b-8df3-30654606a15d.png)


###  Template

![temp-main-program](https://user-images.githubusercontent.com/26597432/197185316-46f142ab-7a1f-4291-a666-0df3008d1cbc.png)